lib/ControlledVocab/tests/ControlledVocab-test.js

```diff
@@ -160,6 +164,22 @@ describe('ControlledVocab', () => {
       expect(cv.cancelButton).to.exist;
     });
 
+    describe('Cancel button is clicked', () => {
+      beforeEach(async () => {
+        await cv.cancelButton().click();
+      });
+
+      it('new button is enabled', () => {
+        expect(cv.disabledNewButton).to.be.false;
+      });
+    });
+  });
+
+  describe('"New" button is clicked', () => {
```

Comment:
Since here you decided to place the name of the button to quotes, please do or don't it to all other places for consistency.